src/index.js

```diff
@@ -1,17 +1,16 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './App'
```

Comment:
Don't you want to set some GlobalStyles like `box-sizing: border-box;`?

Reply:
Done 👍🏾